Makefile

```diff
@@ -32,12 +32,34 @@ configure:
 	cd $(BUILD_DIR) && cmake -G "Unix Makefiles" \
 		-DBUILD_SHARED_LIBS=OFF \
 		-DCMAKE_BUILD_TYPE=Release \
+		-DCOMPILER_RT_ENABLE_MACCATALYST=OFF \
```

Comment:
I've been building with this disabled even before, but now it might not be ok.

The foundamental issue is that xcframework format doesn't support multiple platorms within the same dylib (catalyst and macos):


`error: binaries with multiple platforms are not supported '/Users/josipcavar/Code/rtsan-libs/llvm-project/build/lib/darwin/libclang_rt.rtsan_osx_dynamic.dylib'`

I have two options in mind:
1. Do not suppport Mac Catalyst - easiest
2. Make separate configure target in a Makefile that we will run specifically for xcframework (doubles the work)